src/highdicom/sr/utils.py

```diff
@@ -167,3 +167,29 @@ def get_coded_value(item: Dataset) -> CodedConcept:
             'thus doesn\'t represent a SR Content Item of Value Type CODE.'
         )
     return CodedConcept.from_dataset(value)
+
+
+def is_image(dataset: Dataset) -> bool:
+    """
+    Returns true if the dataset appears to be an image, false otherwise.
+
+    :dcm:`Table C.7-11c Image Pixel Description Macro Attributes <part03/sect_C.7.6.3.html#table_C.7-11c>`
+     Parameters
+    ----------
+    dataset: pydicom.dataset.Dataset
+        Content Item
+    Returns
+        True or False
+    -------
+    """  # noqa: E501
+    if all(key in dataset for key in ('Rows',
+                                      'Columns',
+                                      'SamplesPerPixel',
+                                      'PhotometricInterpretation',
+                                      'BitsAllocated',
+                                      'BitsStored',
+                                      'HighBit',
+                                      'PixelRepresentation')):
+        return True
+    else:
+        return False
```

Comment:
I would probably leave the other TIDs out of scope for now. We can later still add more to the library if this should become necessary.

Reply:
> I did remove BitsStored, Highbit, and PixelRepresentation - but importing from the root highdicom package sets up a circular import. I've left it where it was until we can discuss.

Bummer! In this case, it may be better to keep it as a private function (`_is_image()`) directly in `templates.py` for the time being.

Reply:
Done.